### PR TITLE
add constraint on template store

### PR DESCRIPTION
**Pull request by SOLenG**

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/underscoreTemplateLoader.coffee [hunk 1]
```diff
@@ -37,11 +37,12 @@ do ->
       if !@templates[language][templateName]
         templateLoader = this
         filename = appRouter.generateUrl('loadUnderscoreTemplate')
-        jQuery.get filename, {
+        jqXHR = jQuery.get filename, {
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/public/coffee/underscoreTemplateLoader.coffee]: i think it's more explicit to use ajax property with statuscode method, you have an example in SubmitAdmin.coffee

**Comment by SOLenG** [comment]
it's updated @itkg-alavieille

File: BackofficeBundle/Resources/public/coffee/underscoreTemplateLoader.coffee [hunk 2]
```diff
@@ -37,14 +37,16 @@ do ->
...
-        }, (tpl) ->
-          templateLoader.addTemplate templateName, language, tpl
-          templateLoader.storeTemplates()
-          view.onTemplateLoaded templateName, tpl
-          return
+        jQuery.ajax {
+          method: "GET"
+          url: filename
+          data: { 'language': language,'templateId': templateName }
+          success: (tpl, textStatus, xhr) ->
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/public/coffee/underscoreTemplateLoader.coffee]: you are sure that sucess is only for status code 200 and not for redirect also ?

Comment (SOLenG) [inline BackofficeBundle/Resources/public/coffee/underscoreTemplateLoader.coffee]: effectively, "success" is involved in a number of code http: (status > = 200 && status < 300 || === status 304)

**Comment by itkg-alavieille** [comment]
:+1:

File: BackofficeBundle/Resources/public/coffee/underscoreTemplateLoader.coffee (hunk 2 above)
Comment (nicolasThal) [inline BackofficeBundle/Resources/public/coffee/underscoreTemplateLoader.coffee]: Isn't there a callback directly for the 200 response status

Comment (SOLenG) [inline BackofficeBundle/Resources/public/coffee/underscoreTemplateLoader.coffee]: it existe a callback to the 200 response status, but this solution does not allow me to view the templates in the general case without having to specify all of the answers.

**Pull request merged**